examples/threathunter/threat_intelligence/stix_taxii.py

```diff
@@ -0,0 +1,270 @@
+import logging
+from threatintel import ThreatIntel
+from cabby.exceptions import NoURIProvidedError, ClientException
+from cabby import create_client
+from dataclasses import dataclass, field
+import yaml
+import os
+from stix_parse import parse_stix, BINDING_CHOICES
+from feed_helper import FeedHelper
+from datetime import datetime
+from results import IOC_v2, AnalysisResult
+import urllib3
+
+logging.basicConfig(level=logging.DEBUG)
+
+handled_exceptions = (NoURIProvidedError, ClientException)
+
+
+def load_config_from_file():
+    logging.debug("loading config from file")
+    config_filename = os.path.join(os.path.dirname((os.path.abspath(__file__))), "config.yml")
+    with open(config_filename, "r") as config_file:
+        config_data = yaml.load(config_file, Loader=yaml.SafeLoader)
+        logging.info(f"loaded config data: {config_data}")
+        return config_data
+
+
+@dataclass(eq=True, frozen=True)
+class TaxiiSiteConfig:
+    feed_id: str = ''
+    site: str = ''
+    discovery_path: str = ''
+    collection_management_path: str = ''
+    poll_path: str = ''
+    use_https: bool = False
+    ssl_verify: bool = False
+    cert_file: str = None
+    key_file: str = None
+    default_score: int = 5  # [1,10]
+    username: str = None
+    password: str = None
+    collections: str = '*'
+    start_date: str = '2019-01-01 00:00:00'
```

Comment:
This seems like an odd default start_date?

Reply:
Not sure what a more logical start date would be -- it's kinda arbitrary